<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background: black;</string>
    </property>
    <property name="text">
     <string>Account Balance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background: black;</string>
    </property>
    <property name="text">
     <string>Today Quota:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_balance">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background: black;</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_quota">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background: black;
</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: green;
color: white;</string>
    </property>
    <property name="text">
     <string>Add Income Amount:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>720</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: green;
color: white;</string>
    </property>
    <property name="text">
     <string>Set next Payment Date:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_incomeAmount">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background: black;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushIncomeAmount">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_InNextPaymentDate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background: black;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushNextPaymentDate">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>720</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: red;
color: white;</string>
    </property>
    <property name="text">
     <string>Add Expense Amount:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_ExpenseAmount">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background: black;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushExpenseAmount">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>490</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>720</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushExpenseAmount_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>691</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Recompute Quota</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_OutNextPaymentDate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background: black;
</string>
    </property>
    <property name="text">
     <string>dd/mm/yyyy</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background: black;</string>
    </property>
    <property name="text">
     <string>Next Payment Date:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
